### ISSUE TITLE
Using t-softmax with a pre-trained network

### ISSUE THREAD

**Issue by nantonel** [idiap/tsoftmax#1]
Not really an issue, but this is a short description of a branch with some new experiments.
These are in the branch [tnet](https://github.com/idiap/tsoftmax/tree/tnet) and avoid training NN classifiers from scratch. See readme in that branch for more details.
